### DISCUSSION TITLE
Flexbox Froggy

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#92] [Q&A] [answered]
<img width="919" alt="Screenshot 2023-05-30 102449" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133051222/2092b1e0-ec82-4911-9223-6b581691406e">

I can't figure out how to get the frogs to align to the right on top of the lily pads.

**Comment by bst3ck** [comment]
So I do not want to give the answer away since it is really helpful to work the problem, but here are a couple of suggestions to look at:

1. Your Flex-direction and Flex-wrap can be combined into a single row using flex-flow. (hint: remember there are more items then just **_WRAP_**, maybe try the other option)
2. I will give you one since I think it maybe confusing in the instructions: `align-content: space-between;`. Space-between takes your elements and fills the entire flex container. So this means the first column will be all the way to the left and the last column is all the way to the right.
3. You will need to add one more line to vertically center the yellow frogs. (hint: flex-end will not work to **_CENTER_** your content)

**Reply by Skye404** [reply, marked as answer]
I was able to get it finally. I got the frogs moved over the lily pads, but then it still wanted me to add more elements for some reason. My level 24 didn't have any yellow frogs, and all I was finding online was a level 24 with yellow frogs.

**Reply by bst3ck** [reply]
I am glad you were able to work it out. :)
